bouncer/views.py

```diff
@@ -67,6 +72,7 @@ def index(request):
     append_slash=True,
     renderer='bouncer:templates/notfound.html.jinja2')
 def notfound(request):
+    statsd.incr("views.404.path_did_not_match_any_view_config")
```

Comment:
I wouldn't bother logging the generic 404, as we'll get that from `gunicorn` stats anyway.